src/Collapse/Collapse.component.tsx

```diff
@@ -7,7 +7,6 @@ const EXPANDING = 'expanding';
 const EXPANDED = 'expanded';
 
 export type Props = {
-  isCollapsed?: boolean;
   isOpen?: boolean;
```

Comment:
Since the component named Collapse the input property should be named as 'isCollapsed' or 'isExpanded'

Reply:
Correct. I'll fix it